### DISCUSSION TITLE
有打算支持vuepress吗？

### DISCUSSION THREAD

**Discussion by Jeamorg** [Qexo/Qexo#350] [Ideas]
使用Qexo来写vuepress的文档方便些

**Comment by am-abudu** [comment]
可以考虑，你也可以自己贡献配置文件

**Comment by am-abudu** [comment]
我已经为此写了一个相当潦草的配置文件在Dev分支。你可以体验一下看看有何改进

**Comment by Jeamorg** [comment]
很抱歉，现在才看到。我使用的主题：https://github.com/vuepress-reco/vuepress-theme-reco ，文件夹的结构：https://github.com/JeamUmika/Jeamcc ，部署后的博客：https://jeamcc.pages.dev ，主题方的文件夹规范：http://v2.vuepress-reco.recoluan.com/docs/guide/folder-specification.html

使用了 vuepress 的配置后 Qexo 无论是配置、文章还是页面统统都没显示出来

![图片](https://user-images.githubusercontent.com/93057377/230662186-0a7719f7-5129-48f8-a7e6-303c4bca3e43.png)

**Reply by am-abudu** [reply]
我在Dev版本修复了大量错误，现在应该正确了

**Reply by Jeamorg** [reply]
> 我在Dev版本修复了大量错误，现在应该正确了

现在已经可以显示了，目前没有发现什么bug。不过好像不支持`保存草稿`

**Reply by am-abudu** [reply]
想不出来草稿保存在哪里，而且不能确定文章存储位置
